{"uuid": "bf71da30-daba-4b03-8c06-a3d7f5719e85", "children": ["d7d4daec-a633-4743-87a6-e27b93543f53"], "befores": [{"name": "Conftest", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'get'\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 923, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/_pytest/fixtures.py\", line 783, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/liuchengxu/work/sunxing/sunxing/TestSuite/CcsUserLoginController/conftest.py\", line 21, in Conftest\n    token1=response.get('msg')\n"}, "start": 1582534449687, "stop": 1582534449929}], "start": 1582534449687, "stop": 1582534449990}